app/services/submit_to_app_store.rb

```diff
@@ -30,8 +30,13 @@ def submit(submission)
     # TODO: we only do post requests here as the system is not currently
     # able to support re-sending/submitting an appplication so we can ignore
     # put requests
-    post_manager = AppStoreClient.new
-    post_manager.post(payload)
+    client = AppStoreClient.new
+
+    if submission.is_a?(PriorAuthorityApplication)
+      submission.app_store_updated_at.present? ? client.put(payload) : client.post(payload)
```

Comment:
Oh, and while I remember, you'll probably need to update PriorAuthorityPayloadBuilder to include the further_informations payload (the EventBuilder contains info about the most recent FI, but I think we agreed the payload would contain a history of _all_ FIs

Reply:
ooh sorry glossed over this one and sorted the event part only - will add this in shortly